setup.py

```diff
@@ -34,7 +34,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='2.2.4',
+    version='2.3.0',
```

Comment:
Could you also merge the latest `master` branch and resolve the conflict with the upstream version change in `setup.py`?